### DISCUSSION TITLE
Adding records in bulk

### DISCUSSION THREAD

**Discussion by sumitsharansatsangi** [piccolo-orm/piccolo#611] [General]
Does piccolo supports bulk addition of records in the database?
 I didn't find in the documentation.
Please guide, if it supports or add this feature.

**Comment by dantownsend** [comment]
You can try `insert`. For example:

```python
await MyTable.insert(
    MyTable(my_column=1),
    MyTable(my_column=2),
    MyTable(my_column=3)
)
```
